src/main/java/sh/okx/railswitch/database/RailSwitchDatabase.java

```diff
@@ -15,104 +13,138 @@
 import java.util.concurrent.Executors;
 import java.util.logging.Level;
 import java.util.logging.Logger;
+import org.bukkit.entity.Player;
 
+/**
+ * RailSwitch database class
+ */
 public class RailSwitchDatabase {
-  private static final String DESTINATION_TABLE = "rsw_dest";
-
-  private ConnectionPool pool;
-  private Logger log;
-
-  private String setPlayerDestination;
-  private String getPlayerDestination;
-  private String removePlayerDestination;
-  private ExecutorService service = Executors.newSingleThreadExecutor();
-
-  private final LoadingCache<Player, Optional<String>> destinations = CacheBuilder.newBuilder()
-      .maximumSize(1000)
-      .build(new CacheLoader<Player, Optional<String>>() {
-        @Override
-        public Optional<String> load(Player key) throws Exception {
-          try (Connection connection = pool.getConnection()) {
-            PreparedStatement statement = connection.prepareStatement(getPlayerDestination);
-            statement.setString(1, key.getUniqueId().toString());
-
-            ResultSet resultSet = statement.executeQuery();
-            if (resultSet.next()) {
-              return Optional.of(resultSet.getString("dest"));
-            } else {
-              return Optional.empty();
-            }
-          }
+    
+    private static final String DESTINATION_TABLE = "rsw_dest";
```

Comment:
Not sure why this stuff is made so complicated. It's not like you'd ever want to change the table name and having the sql statements inline makes the code so much easier than this